[2025-08-31 23:43:11,123] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-31T23:43:07.182344+00:00 [queued]>
[2025-08-31 23:43:11,138] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-31T23:43:07.182344+00:00 [queued]>
[2025-08-31 23:43:11,138] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-31 23:43:11,138] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-31 23:43:11,139] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-31 23:43:11,159] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-31 23:43:07.182344+00:00
[2025-08-31 23:43:11,166] {standard_task_runner.py:52} INFO - Started process 1780 to run task
[2025-08-31 23:43:11,171] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'manual__2025-08-31T23:43:07.182344+00:00', '--job-id', '392', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmp_0s9jssk', '--error-file', '/tmp/tmphgy5wm0s']
[2025-08-31 23:43:11,173] {standard_task_runner.py:80} INFO - Job 392: Subtask verify_pipeline
[2025-08-31 23:43:11,250] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline manual__2025-08-31T23:43:07.182344+00:00 [running]> on host 5d7670b258c3
[2025-08-31 23:43:11,362] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-31T23:43:07.182344+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-31T23:43:07.182344+00:00
[2025-08-31 23:43:11,363] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-31 23:43:11,364] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-31 23:43:11,381] {subprocess.py:85} INFO - Output:
[2025-08-31 23:43:11,385] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-31 23:43:41,415] {subprocess.py:92} INFO - yellow open weather_data BQjbDtS5TvO1Q2HrAJGyvA 1 1 95 0 48.7kb 48.7kb
[2025-08-31 23:43:41,418] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-31 23:43:41,485] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250831T234307, start_date=20250831T234311, end_date=20250831T234341
[2025-08-31 23:43:41,530] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-31 23:43:41,591] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
